src/google/protobuf/compiler/cpp/cpp_bootstrap_unittest.cc

```diff
@@ -178,6 +178,18 @@ TEST(BootstrapTest, GeneratedFilesMatch) {
   }
 }
 
+//test Generate in cpp_generator.cc
+TEST(BootstrapTest, optionNotExist) {
```

Comment:
Capitalize the o in option: `optionNotExist` -> `OptionNotExist`

Reply:
DONE